admin/src/test/scala/io/buoyant/admin/LogFormatterTest.scala

```diff
@@ -0,0 +1,34 @@
+package io.buoyant.admin
+
+import java.util.TimeZone
+import java.util.logging.{Level, LogRecord}
+
+import com.twitter.util.TimeFormat
+import org.scalatest.WordSpec
+
+class LogFormatterTest extends WordSpec {
```

Comment:
Note that we use `FunSuite` throughout this repo. I have a minor preference to keep it consistent, but it's in no way a blocker ;)